### PR TITLE
Jakob's Blomer H1 RNTuple benchmarks

### PR DESCRIPTION
**Pull request by oshadura**
In addition, there were introduced an option that allow to download big data files and generate as well H1 dataset.

### CODE REVIEW

File: tools/datafile-generators/h1linkdef.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//
+//  LinkDef.h
+//
+//
+//  Created by Simon Leisibach on 25.09.19.
+//
+
+#ifdef __CINT__
```
Comment (pcanal) [inline tools/datafile-generators/h1linkdef.h]:
```suggestion
#ifdef __ROOTCLING__
```

File: tools/datafile-generators/h1linkdef.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+#pragma link off all globals;
```
Comment (pcanal) [inline tools/datafile-generators/h1linkdef.h]: The link off are no longer needed. (default was switch a long time ago :) )

File: tools/datafile-generators/src/gen_h1.cxx [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
...::move(nelecNTuple), sumcNTuple, /*40-49*/ *sumetc, *yjbc, *Q2jbc, std::move(sumctNTuple), *sumetct, *yjbct, *Q2jbct, *yjbct, *Q2jbct, std::move(pvtx_dNTuple), /*50-59*/ std::move(cpvtx_dNTuple), std::move(pvtx_tNTuple), std::move(cpvtx_tNTuple), *ntrkxy_t, *prbxy_t, *ntrkz_t, *prbz_t, *nds, *rankds, *qds, /*60-69*/ std::move(pds_dNTuple), *ptds_d, *etads_d, *dm_d, *ddm_d, std::move(pds_tNTuple), *dm_t, *ddm_t, *ik, *ipi, /*70-79*/ *ipis, std::move(pd0_dNTuple), *ptd0_d, *etad0_d, *md0_d, *dmd0_d, std::move(pd0_tNTuple), *md0_t, *dmd0_t, std::move(pk_rNTuple), /*80-89*/ std::move(ppi_rNTuple), std::move(pd0_rNTuple), *md0_r, std::move(Vtxd0_rNTuple), std::move(cvtxd0_rNTuple), *dxy_r, *dz_r, *psi_r, *rd0_d, *drd0_d, /*90-99*/ *rpd0_d, *drpd0_d, *rd0_t, *drd0_t, *rpd0_t, *drpd0_t, *rd0_dt, *drd0_dt, *prbr_dt, *prbz_dt, /*100-109*/ *rd0_tt, *drd0_tt, *prbr_tt, *prbz_tt, *ijetd0, *ptr3d0_j, *ptr2d0_j, *ptr3d0_3, *ptr2d0_3, *ptr2d0_2, /*110-134*/ *Mimpds_r, *Mimpbk_r, /* *ntracks,*/ std::move(ntrackNTuple), /*135-143*/ *imu, *imufe, /* *njets,*/ std::move(njetNTuple), /*144-151*/ *thrust, std::move(pthrustNTuple), *thrust2, std::move(pthrust2NTuple), *spher, *aplan, *plan, {nnout[0]}};
```
Comment (pcanal) [inline tools/datafile-generators/src/gen_h1.cxx]: line too long?

Comment (oshadura) [reply tools/datafile-generators/src/gen_h1.cxx]: Thanks!

File: tools/datafile-generators/h1linkdef.h (hunk 2 above)
Comment (oshadura) [reply tools/datafile-generators/h1linkdef.h]: Thanks!!! I didn't know!

File: tools/datafile-generators/h1linkdef.h (hunk 1 above)
Comment (oshadura) [reply tools/datafile-generators/h1linkdef.h]: Done :)

**Comment by oshadura** [comment]
@vgvassilev can you take a look on CMake part?

**Pull request merged**